ios/Podfile.lock

```diff
@@ -477,4 +477,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: a00cfa19ba05b8a86653b23983d6fc5eb45d0925
 
-COCOAPODS: 1.9.2
+COCOAPODS: 1.9.0
```

Comment:
@prehnRA I think this is because I used a different Ruby version from what you used when you first installed. I'd be happy to update the `.tool-versions` file to match you instead, if that makes sense, just let me know which version.

Reply:
That's fine, we can roll with this version.